test/Nancy.Authentication.Basic.Tests/project.json

```diff
@@ -0,0 +1,27 @@
+﻿{
+  "version": "2.0.0-*",
+
+  "description": "Nancy is a lightweight web framework for the .Net platform, inspired by Sinatra. Nancy aim at delivering a low ceremony approach to building light, fast web applications.",

```

Comment:
Let's just kill all these for the tests, including `version`. Makes little sense, easier to maintain :grin: